find/main/routes.py

```diff
@@ -190,7 +196,7 @@ def retrieve_spreadsheet(fund_code: str, submission_id: str):
                 )
             )
 
-        return render_template("find/main/spreadsheet-not-found.html")
+        return render_template("common/templates/common/404.html")
```

Comment:
This will return a 200 status code - the Flask-y way to do this would be to just call `abort(404)` - which is tied up to render the correct error page.

Reply:
Changed.